Topics/Tech_Stacks/Html.md

```diff
@@ -0,0 +1,87 @@
+        ``` {html}
+            <img src="https://yourImageURL.com" alt="My sample image">
+        ```
+#### Video: 
+        Similarly in order to insert a video element in the webpage we use the <video> tag. Here we use a seperate <source> tag to provide the link to the video.  
+        ``` {html}
+            <video>
+                <source src="YourVideo.mp4" type="video/mp4">
+            </video>
+        ```
+#### Table:
+        In order to display a table we use the <table> tag. Inside the table tags we use <thead> tag for the title row, the <tbody> tag for all the information rows and the <tfoot> tag for the concluding row. Inside each of these tags the <tr> tag is used to start a row and <td> for the column in it. The <th> tag is used for the header column inside the <tr> tags and <td> inside the <tr> tags in the <tfoot> section.  
+        ``` {html}
+            <table>
+                <thead>
+                    <tr>
+                        <th>Column 1</th>
+                        <th>Column 2</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>element at row 1 column 1</td>
+                        <td>element at row 1 column 2</td>
+                    </tr>
+                    <tr>
+                        <td>element at row 2 column 1</td>
+                        <td>element at row 2 column 2</td>
+                    </tr>
+                </tbody>
+                <tfoot>
+                    <tr>
+                        <td>footer column 1</td>
+                        <td>footer column 2</td>
+                    </tr>
+                </tfoot>
+            </table>
+        ```
+#### Links:
+        The <a> tag is used to provide links to external websites with the href attribute pointing to the desired webpage.  
+#### Input:
+        The <input> tag is used to get information from the user. The type attribute in the tag determines the format of the input. Some types are text, time, date, image and email.  
+#### Div:
+        The <div> tag creates a section in the document. An empty div tag will have size 0. The size of the tag can be manipulated in css or automatically takes the size of the contents in it.
+#### Lists:
+        The <ul> tag is used to create lists. Inside the <ul> we use <li> tags for each list element. For example:  
+         ``` {html}
+            <ul>
+                <li> List element 1</li>
+                <li> List element 2</li>
+            </ul>
+        ```
+### Usage of elements:
+    Elements can be nested within each other. For example a <p> may contain text, followed by a table and a list if needed. A <p> tag could contain another <p> tag in it as well!  Elemnts will be rendered in the order in which they are coded.  \
+### Styling:
+    In order to customise the look of elements html is combined with css to make the webpages look cleaner.
```

Comment:
Please specify a little more information on how to combine styling with html elements